submissions/Cashass/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <title>Popup-menu</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+
+  <body>
+    <div class="wraper">
+      <header class="header">
+        <div class="container">
+          <nav class="header__menu">
+            <ul class="header__lists">
+              <li class="header__list">
+                <a href="#" class="header__link">Mail</a>
+              </li>
+              <li class="header__list">
+                <a href="#" class="header__link">Images</a>
+              </li>
+              <li class="header__list header__link">
+                <input
+                  type="checkbox"
+                  class="header__input"
+                  id="header__input"
+                />
+                <label for="header__input" class="header__input-label">
+                  <img
+                    src="icon/menu.png"
+                    alt="menu"
+                    class="header__logo-menu"
+                  />
+                </label>
+                <div class="popup">
+                  <ul class="header__lists-popup">
+                    <li class="popup__list">
+                      <a href="#" class="popup__link">
+                        <img
+                          class="popup-img"
+                          src="icon/Безпека.png"
+                          alt="Безпека"
+                        />Безпека</a
+                      >
+                    </li>
+                    <li class="popup__list">
+                      <a href="#" class="popup__link">
+                        <img
+                          class="popup-img"
+                          src="icon/Календар.png"
+                          alt="Календар"
+                        />Календар</a
+                      >
```

Comment:
Read how and when to populate alt attribute value [6 simple rules for good alt-text](https://habr.com/ru/company/htmlacademy/blog/667540/)

Reply:
I almost don’t understand, it says what not to write near decorative icons, and the text should be meaningful. Thank you.

Reply:
Should I remove the entire description, leave only an empty alt?

Reply:
> I almost don’t understand, it says what not to write near decorative icons, and the text should be meaningful. Thank you.

Make an alt description if the picture is relevant to the content. For decorative images, use alt, which is equal to an empty string

Reply:
Make an alt description if the picture is relevant to the content. For decorative images, use alt, which is equal to an empty string